src/woo_publications/publications/models.py

```diff
@@ -321,6 +335,44 @@ def clean(self):
                 )
             )
 
+    def save(self, *args, **kwargs) -> None:
+        if not self.vanaf:
+            if not self.registratiedatum:
+                self.registratiedatum = timezone.now()
+            self.vanaf = self.registratiedatum
```

Comment:
`registratiedatum` is always automatically set by django when the record is created because of the `auto_now_add`. All of this is not needed :)

Reply:
I used this so that vanaf could be the exact same as registratiedatum instead of `self.vanaf = timezone.now()`.
Because before the document is created registratiedatum is still None.